src/lib/foundation/Chooser.svelte

```diff
@@ -2,6 +2,7 @@
     import { createEventDispatcher } from "svelte";
     import { t, locale } from "$lib/language/translate";
     import { defaultFilterCategory } from "../filters";
+    import CloseIcon from "$lib/icons/close.svg";
```

Comment:
Great work and thanks for doing this!

Couple small things...

I retargeted to the "dev" branch which is something new we're going to try. Makes it easier to merge in multiple features/fixes under one PR before deployment.

**To do**
- Pull latest from dev
- Use the existing x-mark icon (I recently added this when I implemented the "How to Borrow" modal)